demo/app/components/chart/chart.component.ts

```diff
@@ -68,25 +72,25 @@ export class ChartComponent {
     if (tsChartXYTypeCheck(chart)) {
       chart.data = XY_DATA;
 
-      const dateAxis = chart.xAxes.push(new am4charts.DateAxis());
+      const dateAxis = chart.xAxes.push(new am4charts.DateAxis() as any);
```

Comment:
Yeah, that's why I had to make these changes. Strict mode didn't exist when amCharts v4 was written so they are working on making it fully compliant - but some holes still exist.

I opted to let consumers kill typings rather than disabling 'strict' for everything.